charts/skypilot/values.yaml

```diff
@@ -198,12 +169,21 @@ rbac:
     - apiGroups: [ "networking.k8s.io" ]
       resources: [ "ingressclasses" ]
       verbs: [ "get", "list", "watch" ]
-    # TODO(aylei): This is too permissive, consider passing existing roles instead.
-    # Required for granting cluster permissions to SkyPilot Pods.
-    - apiGroups: [ "rbac.authorization.k8s.io" ]
-      resources: [ "clusterroles", "clusterrolebindings" ]
-      verbs: [ "get", "list", "watch", "create", "update", "patch", "delete" ]
-
+    # Required for accessing ingress service.
+    # TODO(aylei): unify the ingress of API server and task pods, then remove this.
```

Comment:
Follow up here: https://github.com/skypilot-org/skypilot/issues/4915

Reply:
I did not separate this permission to `ingress-nginx` namespace for the following because the namespace is hardcoded now. If we enable this permission by default, the installation will fail if `ingress-nginx` namespace is not exist. If we disable it by default, users will need additional steps to tune the permission. Plus this permission will be refactored soon in https://github.com/skypilot-org/skypilot/issues/4915, here we just keep it simple.